packages/transformers/vue/src/VueTransformer.js

```diff
@@ -394,7 +390,7 @@ ${
           let styleAsset = {
             type: 'css',
             content: styleComp.code,
-            sideEffects: !style.module,
+            sideEffects: true,
```

Comment:
Not sure if this is correct...

Reply:
👍  I can't think of a case where a CSS asset should be ever have `sideEffects: false`.